src/nvim/lua/executor.c

```diff
@@ -1800,3 +1801,34 @@ void nlua_execute_log_keystroke(int c)
 #endif
 }
 
+// Get current scripts indentifier
+sctx_T *nlua_get_sctx(void)
+{
+  lua_State *const lstate = nlua_enter();
+  sctx_T *retval = (sctx_T *)xmalloc(sizeof(sctx_T));
+  retval->sc_seq = -1;
+  retval->sc_sid = SID_LUA;
+  retval->sc_lnum = -1;
+  lua_Debug *info = (lua_Debug *)xmalloc(sizeof(lua_Debug));
+  char *runtime_path = vim_getenv("VIMRUNTIME");
```

Comment:
use an internal variable for this.

Reply:
I was actually wondering if we have stored this value in some global . I tried greping some guesses didn't find one . Do you know what the internal variable is ?

Reply:
Seems it doesn't exist yet, so let's use this for now.

Reply:
Alright . Also do you know why $VIMRUNTIME on windows has '/' as path separator instead of '\' ? Is this intentional or a bug ?